src/modules/logger.js

```diff
@@ -0,0 +1,205 @@
+const {remote} = require('electron')
+const fs = require('fs');
+const setting = remote.require('./setting')
+const dialog = require('./dialog')
+const winston = require('winston');
+const path = require('path')
+
+let logger = null
+let logFileNameGTP = null
+let staleLogFilePath = false
+
+exports.writeToLogFileGTP = function(enginestream) {
```

Comment:
First of all, I would like to clean up this module by renaming lengthy function names like this one to something shorter. Probably `write` will even suffice here, since this module is all about logging GTP data from the console. In light of this, it's probably best to rename `logger` into `gtplogger`.

Reply:
Let me know if the new names are OK. The new commit 6b159c6 should resolve this and all all of the above requested changes.